be/functions/src/tests/test-check-user-rewards.sh

```
@@ -0,0 +1,28 @@
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+FUNCTIONS_DIR="$(cd "$SCRIPT_DIR/../.." && pwd)"
+
+USER_ID="${1:-reward-test-user-001}"
+
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+echo "📊 사용자 리워드 확인 테스트"
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+echo ""
+
+cd "$FUNCTIONS_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)